bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/HueBridge.java

```diff
@@ -425,8 +426,6 @@ public String setLightName(HueObject light, String name) throws IOException, Api
      * @return all lights pseudo group
      */
     public Group getAllGroup() {
-        requireAuthentication();
```

Comment:
Was it there really for no valid reason ?
This is requiring a good analysis.

Reply:
The deep analysis will reveal that this method never has been used before.